.travis.yml

```diff
@@ -2,14 +2,13 @@ sudo: false
 language: scala
 script:
   - sbt ++$TRAVIS_SCALA_VERSION root/scalastyle testSuiteJVM/scalastyle testSuiteJVM/test:scalastyle testSuite/scalastyle testSuite/test:scalastyle
-  - sbt ++$TRAVIS_SCALA_VERSION testSuiteJVM/test
-  - sbt ++$TRAVIS_SCALA_VERSION testSuite/test
-  - sbt ++$TRAVIS_SCALA_VERSION 'set scalaJSUseRhino in Global := false' testSuite/test
-  - sbt ++$TRAVIS_SCALA_VERSION 'set scalaJSUseRhino in Global := false' 'set scalaJSStage in Global := FullOptStage' testSuite/test
-  - sbt ++$TRAVIS_SCALA_VERSION publishLocal
+  - sbt ++$TRAVIS_SCALA_VERSION +testSuiteJVM/test
+  - sbt ++$TRAVIS_SCALA_VERSION +testSuite/test
+  - sbt ++$TRAVIS_SCALA_VERSION 'set scalaJSStage in Global := FullOptStage' +testSuite/test
+  - sbt ++$TRAVIS_SCALA_VERSION +publishLocal
```

Comment:
Do not use `+` in Travis scripts. The cross-testing is achieved by Travis' matrix.